kong/tracing/instrumentation.lua

```diff
@@ -201,7 +211,8 @@ function _M.request(ctx)
     http_flavor = string.format("%.1f", http_flavor)
   end
 
-  local active_span = tracer.start_span(span_name, {
+  local active_span = tracer.start_span("API_gateway", {
```

Comment:
am I right that this change means otel will lose the "path" information?
Would it make sense to pass `method` and `path` as `attributes` instead so they could be then used by DD to make the `resource` field, while otel wouldn't lose the information?